woocommerce/woocommerce-ios
# Google Ads: Update dashboard card layout

Closes: #13302 


## Description

This PR updates the Google Ads dashboard card layout following the feedback from Google. Changes include:
- Update the stats to display the total stats for paid campaigns instead of just the last active campaign.
- Display "Paid campaign performance" in the header instead of the last campaign's name and status.
- Update the title on the empty state to be the subtitle used in Hub Menu item for consistency.

## Testing steps

- Pre-requisite: Install, activate, and set up the Google Ads plugin to a test site following pe5sF9-2Qo-p2. Ensure that you're using the plugin version 2.7.5 or above. Remove any existing campaigns on your store.
- Build the app and log in to your store.
- Enable Google Ads card on the dashboard if you haven't already.
- Confirm that the title on the empty state of the card matches the subtitle on the Hub Menu entry point.
- Create a new campaign from the dashboard card.
- After successfully creating a campaign, confirm that the dashboard card displays the new title and the total stats for all campaigns. Since test campaigns don't have any impressions or clicks, feel free to stub the response of `GET wc/gla/ads/reports/programs` to confirm the correct stats being displayed.


## Screenshots

State | Before | After
--- | --- | ---
Empty | <img src="https://github.com/user-attachments/assets/62975940-a1c9-4709-8c57-59dab1f98c59" width=320 /> | <img src="https://github.com/user-attachments/assets/e5b6137e-d830-4af8-a724-7a328528abab" width=320 />
Data | <img src="https://github.com/user-attachments/assets/20ba35ee-0b0d-49f7-9660-7813d0b2b8bc" width=320 /> | <img src="https://github.com/user-attachments/assets/c285e4ef-de5f-48a6-80b8-99d371e2fbc9" width=320 />


---
